### ISSUE TITLE
Include short-term effects in sizing

### ISSUE THREAD

**Issue by wouterpeere** [wouterpeere/GHEtool#26]
Right now, GHEtool ignores the internal capacity of the borehole, by making use of the equivalent borehole thermal resistance. This short-term effects however, can play an important role in some cases.

A resistance-capacitance model can be implemented like the one from [(Minaei and Maerefat, 2017)](https://doi.org/10.1016/j.enbuild.2017.04.064).

**Comment by wouterpeere** [comment]
A first short-term effect (cylindrical correction) was added in issue #187

**Comment by LoneMeertens** [comment]
Next steps should focus on the following improvements:
- [ ]  Develop a more general method for determining the far field radius that is applicable to non-rectangular shapes.
- [ ] Calculate the fluid factor based on the borehole configuration.

The sensitivity analysis should consider the following parameters:
- [ ] Borehole spacing
- [ ] The num_intervals parameter in the numerical model.

**Comment by wouterpeere** [comment]
Hi @LoneMeertens,

Perhaps it is best to develop all the tasks you mentioned above before merging into the main branch? What do you think?

Best,
@wouterpeere

**Comment by LoneMeertens** [comment]
@wouterpeere I completely agree that the method must be robust and thoroughly validated for both single and double-U tubes before merging with the main branch. The extension to include sizing with double-U tube boreholes is already in place but has yet to be validated. I will keep you informed once the full validation is completed and the tasks mentioned above are implemented.
